docker/Dockerfile

```diff
@@ -1,16 +1,38 @@
-FROM ubuntu:20.04
+FROM golang:alpine as gobuild
 
-# Make sure that the underlying container is patched to the latest versions
-RUN apt-get update && \
-    apt-get install -y gzip tar wget
+RUN apk update && \
+    apk add ca-certificates gcc git make musl-dev && \
+    git clone https://github.com/mattermost/focalboard && \
```

Comment:
Correct me if I'm wrong but won't this always grab the `main` branch when building the docker image? So when we tag and create a docker image it would create the image from master and not the tag. Same issue when building from a branch

I think we need to pass the branch/tag in as an argument and do a `git checkout $TAG` after the clone, or do a COPY instead of a git clone

Reply:
Yes, that's true. I'll change this; how about using a `ARG` and specify this while building with `--build-arg TAG=v0.6.1`?

Reply:
Per https://docs.docker.com/docker-hub/builds/advanced/ we can use `$SOURCE_BRANCH` for the dockerhub builds

Reply:
I think this should not do a git clone, but copy the local files. This would allow for easily modifying something and re-building it, and people will most likely have the repository cloned when building anyway, at which point cloning it again is counterintuitive.